packages/react-api/src/Api.tsx

```diff
@@ -183,15 +183,29 @@ async function loadOnReady (api: ApiPromise, endpoint: LinkOption | null, inject
   };
 }
 
+function getWellKnownChain (chain = 'polkadot') {
+  if (chain === 'kusama') {
+    return WellKnownChain.ksmcc3;
+  }
+
+  if (chain === 'westend') {
+    return WellKnownChain.westend2;
+  }
+
+  if (chain === 'rococo') {
+    return WellKnownChain.rococo_v2_1;
+  }
+
+  return WellKnownChain.polkadot;
+}
+
 // eslint-disable-next-line @typescript-eslint/require-await
 async function createApi (apiUrl: string, signer: ApiSigner, onError: (error: unknown) => void): Promise<Record<string, Record<string, string>>> {
   const types = getDevTypes();
   const isLight = apiUrl.startsWith('light://');
 
   try {
-    const provider = isLight
-      ? new ScProvider(apiUrl.replace('light://substrate-connect/', '') as 'polkadot')
-      : new WsProvider(apiUrl);
+    const provider = isLight ? new ScProvider(getWellKnownChain(apiUrl.replace('light://substrate-connect/', ''))) : new WsProvider(apiUrl);
```

Comment:
```suggestion
    const provider = isLight
      ? new ScProvider(getWellKnownChain(apiUrl.replace('light://substrate-connect/', '')))
      : new WsProvider(apiUrl);
```

Just prefer is slightly spaced, readability is in the eye of the beerholder :)